website/views.py

```diff
@@ -3711,6 +3711,40 @@ def invite_friend(request):
     }
     return render(request, "invite_friend.html", context)
 
+def trademark_search(request, **kwargs):
+    print(request.method)
+    if request.method == "POST":
+        slug = request.POST.get("query")
+        return redirect("trademark_detailview", slug=slug)
+    return render(request, "trademark_search.html")
+
+
+def trademark_detailview(request, slug):
+    if settings.USPTO_API is None:
+        return HttpResponse("API KEY NOT SETUP")
+    
+    trademarkAvailable_url = "https://uspto-trademark.p.rapidapi.com/v1/trademarkAvailable/%s" % (
```

Comment:
Here and below: this must be snake-cased.

Reply:
you mean Trademark_Available_Url and Trademark_Available_Response

Reply:
`trademark_available_url`

Reply:
Done 👍

Reply:
I don't think it's done -- you have a mix of camel/snake case currently. https://www.pluralsight.com/blog/software-development/programming-naming-conventions-explained

Reply:
@arkid15r Ohkay got that !
Thankyou could you please see if this was required ?

Reply:
Yes, that was it 👍